manifests/apache.pp

```diff
@@ -50,7 +50,6 @@
   $api_default_request_headers = [
     "unset ${remote_user_environ_header}",
     "unset ${remote_user_environ_header_underscore}",
-    "set ${remote_user_environ_header} \"%{SSL_CLIENT_S_DN_CN}s\" env=SSL_CLIENT_S_DN_CN",
```

Comment:
Technically, this is a breaking change. At the same time, this never worked so I'd err on the side of "enhancement"?

Reply:
lol, OUR TESTS WERE USING THIS?!

Reply:
I have a vague recollection that we used to create the certs with `cn=admin` and the acceptance tests copied that. When we changed it in certs it wasn't reflected here

Reply:
I did just mark it as backwards incompatible just so it's clear we're changing behavior. Recently we had some cherry picks with bumps and I don't want to accidentally include this in an existing release.